test/unit/xml_description_test.py

```diff
@@ -3,22 +3,42 @@
 
 from .test_helper import *
 
-from kiwi.exceptions import (
-    KiwiSchemaImportError,
-    KiwiValidationError,
-    KiwiDescriptionInvalid,
-    KiwiDataStructureError
-)
+from kiwi.exceptions import *
```

Comment:
Please don't. Stars are _very bad_ style and [PEP8 hates that](https://www.python.org/dev/peps/pep-0008/#imports).